Warpi/line.py

```diff
@@ -0,0 +1,20 @@
+# Importing turtle module
+from turtle import *
```

Comment:
Per the pep 8 style guide there should be two blank lines between your imports and your first line of code https://realpython.com/python-pep8/#blank-lines